### ISSUE TITLE
Add String.EqualsIgnoreCase() to Z.Core

### ISSUE THREAD

**Issue by adamfisher** [zzzprojects/Z.ExtensionMethods#12]
The following is a proposal to add `Z.ExtensionMethods/src/Z.Core/System.String/String.EqualsIgnoreCase.cs` which would contain the following convenience extension method. I miss it from my Java days and the .NET version is too verbose/cumbersome.

```csharp
public static bool EqualsIgnoreCase(this string @string, string otherString);
```

**Comment by JonathanMagnan** [comment]
Hello @adamfisher ,

Thank you for your proposition, I will add it to our to-do list.

Best Regards,

Jonathan

---

**Performance Libraries**
`context.BulkInsert(list, options => options.BatchSize = 1000);  `
[Entity Framework Extensions](http://entityframework-extensions.net/) • [Bulk Operations](http://bulk-operations.net/) •  [Dapper Plus](http://dapper-plus.net/) •  [LinqToSql Plus](http://linqtosql-plus.net/)

**Runtime Evaluation**
`Eval.Execute("x + y", new {x = 1, y = 2}); // return 3`
[C# Eval Function](http://eval-expression.net/) • [SQL Eval Function](http://eval-sql.net/)

**Comment by falwickster** [comment]
Hello @adamfisher @JonathanMagnan 

I've added this simple functionality. It's already in my pull request.

EDIT: Although I am not sure if it should be in a seperate pull request.

Best regards,
Chris

**Comment by JonathanMagnan** [comment]
Hello @falwickster ,

No, you should not import other pull in yours.

For example, if this one get refused for a reason X, I will be unable to merge your changes now when previously, it was independent of this pull.

Best Regards,

Jonathan

**Comment by adamfisher** [comment]
Thanks for PR https://github.com/zzzprojects/Z.ExtensionMethods/pull/14 which will eliminate the need for me to always define it. Hopefully it gets approved.

**Comment by JonathanMagnan** [comment]
Hello @adamfisher ,

The pull from @falwickster has been merged in the v2.1.0

Best Regards,

Jonathan

**Comment by JonathanMagnan** [comment]
Hello @adamfisher ,

This issue will be closed since it has been completed.

Feel free to reopen it if you feel otherwise.

Best Regards,

Jonathan
